src/server/index.jsx

```diff
@@ -78,17 +78,16 @@ if (process.env.APP_ENV !== 'local') {
 /*
  * Local env routes - fixture data
  */
+const sendDataFile = (res, dataFilePath, next) => {
```

Comment:
I think moving this function outside of the `if (process.env.APP_ENV === 'local') {` on line 90 would mean we start serving data for other routes (articles, front pages etc) in production which we shouldn't do

Reply:
@pjlee11 are you sure?  the other routes are still inside the `if` block so shouldn't be registered

This change is just lifting the definition of this helper function to so it is accessible to both the `if env === local` and the 'if env === local || test` blocks - but it's only ever being called inside those blocks

Reply:
Ahhh yeah you're right, I misread the changes. My comment above is moot